### PR TITLE
Add kotlinx json serialization support

### LINKED ISSUE #3: Add new serializer - kotlin.serialization JSON

### PR DESCRIPTION
**Pull request by enzo-beltrami**
Closes #3

### CODE REVIEW

File: serializer/kotlinx-json-serializer/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                )
+            }
+        }
+    }
+
+    sourceSets {
+        val commonMain by getting{
+            dependencies {
+                implementation(project(Kached.core))
+                implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.0.1")
```
Comment (faogustavo) [inline serializer/kotlinx-json-serializer/build.gradle.kts]: Please, add this to the Dependencies.kt file

**Review by faogustavo — CHANGES_REQUESTED** [review]
Nice man, thanks.
Just fix the KTLint and extract the hardcoded library to the dependencies file please

**Comment by enzo-beltrami** [comment]
> Nice man, thanks.
> Just fix the KTLint and extract the hardcoded library to the dependencies file please

Just fixed it, however there's still an error on Gradle and I don't know how to fix it :thinking: 

```
Could not create an instance of type org.jetbrains.kotlin.gradle.targets.js.subtargets.KotlinBrowserJs.
```

**Comment by faogustavo** [comment]
@enzo-beltrami I've just commented the JS setup in the Gradle file and after the merge, I'll investigate this

**Review by faogustavo — APPROVED** [review]

**Pull request merged**